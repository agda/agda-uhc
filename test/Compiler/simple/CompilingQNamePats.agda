
open import Common.Prelude
open import Common.Reflection

postulate
  X Y : Set

isX : QName â†’ Bool
isX (quote X) = true
isX _ = false


main : IO Unit
main = putStrLn ((if isX (quote X) then "yes" else "no") +S+
                 (if isX (quote Y) then "yes" else "no"))
