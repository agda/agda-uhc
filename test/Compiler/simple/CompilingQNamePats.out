EXECUTED_PROGRAM

ret > ExitSuccess
out > yesno
out >
